doc/source/notebooks/03_node_properties.ipynb

```diff
@@ -186,14 +186,12 @@
       "text/plain": [
-       "<Figure size 432x288 with 1 Axes>"
+       "<Figure size 640x480 with 1 Axes>"
```

Comment:
Just wondering, why images are now 640x480 instead of 432x288?

Reply:
Not sure. I mean the default seems to have been 640x480 for a while:
https://matplotlib.org/stable/users/prev_whats_new/dflt_style_changes.html#figure-size-font-size-and-screen-dpi

So the question is more like why was it 432x288?